commit c688437d2f193a66d863d7c2c45094d29abf879a
Author: hasufell <julian.ospald@googlemail.com>
Date:   Mon Mar 9 15:23:07 2015 +0300

    relocate executables sound-convert and s-c_resample which get installed
    into datadir by default

diff --git a/libutil/src/files.h b/libutil/src/files.h
index 3856f17..8ce89e2 100644
--- a/libutil/src/files.h
+++ b/libutil/src/files.h
@@ -140,7 +140,7 @@ const char* const FILE_PATHS[] =
     /* 54 */RTTRDIR "",                               // unbenutzt
     /* 55 */SETTINGSDIR "/LSTS/SOUND.LST",            // Die konvertierte sound.lst
     /* 56 */RTTRDIR "/sound.scs",                     // Das konvertier-script
-    /* 57 */RTTRDIR "",                               // Basispfad fürn Soundconverter
+    /* 57 */DRIVERDIR "",                             // Basispfad fürn Soundconverter
     /* 58 */GAMEDIR "/DATA/MIS0BOBS.LST",
     /* 59 */GAMEDIR "/DATA/MIS1BOBS.LST",
     /* 60 */GAMEDIR "/DATA/MIS2BOBS.LST",
